ehitchhiking-ui/src/app/shared/components/trips/trips.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="trips-modal-container">
+  <header class = "form-row">
+    <h1 mat-dialog-title class = "trips-title">History</h1>
+    <i class="material-icons">
+      history
+    </i>
+  </header>
+  <div mat-dialog-content class="trips"
+       >
+    <div
+      class="trips-user"
+      *ngFor="let trip of tripsArray; trackBy: trackByFn"
+    >
+      <app-trip class = "trip" [trip]=trip (change)="tripChange($event)"></app-trip>
+    </div>
+
+      <app-no-data *ngIf="!tripsArray.length"></app-no-data>
+
+<!--    <div class = "form" *ngIf="!tripsArray.length">-->
+<!--      <span class="center" >List is empty!</span>-->
+<!--    </div>-->
+
+  </div>
+  <footer>
+    <div mat-dialog-actions class="trips-btn">
+      <button mat-button (click)="exit()">Back</button>
+      <button mat-button (click) = "replase_all()" *ngIf="tripsArray.length">Replase all</button>
+      <button mat-button (click) = "downloadMore()" *ngIf="tripsArray.length">More</button>
```

Comment:
You should disable button if all trips are visible

Reply:
fixed